chain-spec-generator/Cargo.toml

```
@@ -0,0 +1,50 @@
+[package]
+name = "chain-spec-generator"
+version.workspace = true
+authors.workspace = true
+edition.workspace = true
+repository.workspace = true
+license.workspace = true
+
+[dependencies]
+clap = { version = "4.4.4", features = [ "derive" ] }
+hex-literal = "0.4.1"
+serde_json = "1.0.107"
+serde = { version = "1.0.188", features = ["derive"] }
+
+polkadot-runtime = { path = "../relay/polkadot" }
+polkadot-runtime-constants = { path = "../relay/polkadot/constants" }
+kusama-runtime = { package = "staging-kusama-runtime", path = "../relay/kusama" }
+kusama-runtime-constants = { path = "../relay/kusama/constants" }
+
+sc-chain-spec = "24.0.0"
+polkadot-runtime-parachains = "4.0.0"
+polkadot-primitives = "4.0.0"
+sp-consensus-babe = "0.29.0"
+sp-authority-discovery = "23.0.0"
+sp-core = "25.0.0"
+pallet-staking = "25.0.0"
+sc-consensus-grandpa = "0.16.0"
+pallet-im-online = "24.0.0"
+sp-runtime = "28.0.0"
+sp-consensus-beefy = "10.0.0"
+xcm = { package = "staging-xcm", version = "4.0.0" }
+parachains-common = { version = "4.0.0" }
+cumulus-primitives-core = { version = "0.4.0" }
+
+asset-hub-polkadot-runtime = { path = "../system-parachains/asset-hubs/asset-hub-polkadot" }
+asset-hub-kusama-runtime = { path = "../system-parachains/asset-hubs/asset-hub-kusama" }
+collectives-polkadot-runtime = { path = "../system-parachains/collectives/collectives-polkadot" }
+bridge-hub-polkadot-runtime = { path = "../system-parachains/bridge-hubs/bridge-hub-polkadot" }
+bridge-hub-kusama-runtime = { path = "../system-parachains/bridge-hubs/bridge-hub-kusama" }
+
+[features]
+runtime-benchmarks = [
+	"asset-hub-polkadot-runtime/runtime-benchmarks",
+	"asset-hub-kusama-runtime/runtime-benchmarks",
+	"bridge-hub-polkadot-runtime/runtime-benchmarks",
+	"bridge-hub-kusama-runtime/runtime-benchmarks",
+	"collectives-polkadot-runtime/runtime-benchmarks",
+	"kusama-runtime/runtime-benchmarks",
+	"polkadot-runtime/runtime-benchmarks",
```

```suggestion
	"asset-hub-polkadot-runtime/runtime-benchmarks",
	"asset-hub-kusama-runtime/runtime-benchmarks",
	"bridge-hub-polkadot-runtime/runtime-benchmarks",
	"bridge-hub-kusama-runtime/runtime-benchmarks",
	"collectives-polkadot-runtime/runtime-benchmarks",
	"encointer-kusama-runtime//runtime-benchmarks",
	"glutton-kusama-runtime/runtime-benchmarks",
	"kusama-runtime/runtime-benchmarks",
	"polkadot-runtime/runtime-benchmarks",
```

we should add `encointer` and `glutton` here